doc/install/index_manual.rst

```diff
@@ -69,6 +68,4 @@ sequence.
 .. toctree::
    :maxdepth: 2
 
-   Upgrading Ceph <upgrading-ceph>
```

Comment:
Which is correct: we don't have a documentation to manually upgrade ceph. The link was misleading.